### ISSUE TITLE
Getting blank page in local env

### ISSUE THREAD

**Issue by albertoalmansa-e2x** [bigcommerce/b2b-buyer-portal#14]
Hello,

I have checked out the code and tried to run it locally, but I am getting a blank page that just says "Loading...", after I click on "Login". 

These are the screenshots:

<img width="1921" alt="Screenshot 2024-05-20 at 12 58 05" src="https://github.com/bigcommerce/b2b-buyer-portal/assets/43065155/06142529-44b8-448f-bc77-5caecb87b66a">

<img width="1927" alt="Screenshot 2024-05-20 at 12 58 14" src="https://github.com/bigcommerce/b2b-buyer-portal/assets/43065155/d94df09b-6210-487f-b11c-c351ff7a7a0e">

In my env file I am pointing to the same client_id that is in the example env file.

Can you please advise?

Thanks.

**Comment by bc-marco** [comment]
Hello @albertoalmansa-e2x 
Currently we have identified 2 ways to access to buyer portal:
1. By using you bigcommerce store url (https://store-{storeHash}-{channelId}.mybigcommerce.com) or cname url (mystore.com),
2. By using stencil cli tool

Both will work after you configure the script tags on your store configuration

**Comment by albertoalmansa-e2x** [comment]
Hello @bc-marco ,

But I am trying to run or locally following the readme.

Thanks.

**Comment by bc-marco** [comment]
> Hello @bc-marco ,
> 
> But I am trying to run it locally following the readme.
> 
> Thanks.

Yes, but in order to consume the bundled files, you should be using the store, not the local development direction on your browser

PS. Stencil CLI is not fully working yet, we're working on a fix for that

**Comment by albertoalmansa-e2x** [comment]
Thank you @bc-marco , that worked. I can now see it integrated with cornerstone and hit the buyer-portal that runs on my local env. I hadn't imagined it had to be through the deployed theme.

Kind regards,
Alberto.
